docs/CONTRIBUTING.md

```diff
@@ -107,6 +107,25 @@ reproduce it, the faster we can fix the bug! It's also helpful to have the
 output of `enso --version`, as that will let us know if the bug is Operating
 System or Architecture specific.
 
+### Turning on verbose logs
+
+Sometimes, it is helpful to attach a verbose log to your bug report. The way to
+enable verbose logging depends on which version of Enso you are using. For a
+standalone distribution (`.exe` on Windows, `.AppImage` on Linux), you can
+enable verbose logging by passing `-debug.verbose` option. If you are starting
+the `project-manager`, or language server separately, then pass
+`--log-level trace` option. With verbose logging, there are a lot of messages
+printed to the standard output, and it is possible that on slower terminal
+emulators this will clog the terminal and hence the whole backend. To avoid
+this, we recommend redirecting the output to `/dev/null`, via a command like
+`enso -debug.verbose > /dev/null 2>&1`.
+
+The logs are kept in a central location - on Linux, they are in
+`$XDG_DATA_HOME/enso/log` ( usually `~/.local/share/enso/log`), and on Windows
+they are in `%APPDATA%\enso\log`. The log level name consists of the timestamp
```

Comment:
```suggestion
The logs are kept in a central location `$ENSO_DATA_DIRECTORY/log` - on Linux,
they are in `$XDG_DATA_HOME/enso/log` ( usually `~/.local/share/enso/log`), and
on Windows they are in `%APPDATA%\enso\log`, see
[distribution.md](https://github.com/enso-org/enso/blob/develop/docs/distribution/distribution.md)
for details. The log level name consists of the timestamp
```

Reply:
Technically the location is described as `ENSO_DATA_DIRECTORY/log` https://github.com/enso-org/enso/blob/develop/docs/distribution/distribution.md

Reply:
Also, I'm not sure if the suggestion above is properly formatted

Reply:
I have run `./run fmt`, so it should be OK now.